static/gsAdmin/components/customers/customerOverview.tsx

```diff
@@ -438,19 +438,15 @@ function CustomerOverview({customer, onAction, organization}: Props) {
   );
   const region = regionMap[organization.links.regionUrl] ?? '??';
 
-  const productTrialCategories = customer.canSelfServe
-    ? Object.values(BILLED_DATA_CATEGORY_INFO).filter(
-        categoryInfo =>
-          categoryInfo.canProductTrial &&
-          customer.planDetails.categories.includes(categoryInfo.plural)
-      )
-    : [];
-
-  const productTrialCategoryGroups = customer.canSelfServe
-    ? Object.values(customer.planDetails.availableReservedBudgetTypes).filter(
-        group => group.canProductTrial
-      )
-    : [];
+  const productTrialCategories = Object.values(BILLED_DATA_CATEGORY_INFO).filter(
+    categoryInfo =>
+      categoryInfo.canProductTrial &&
+      customer.planDetails?.categories.includes(categoryInfo.plural)
+  );
+
+  const productTrialCategoryGroups = Object.values(
+    customer.planDetails?.availableReservedBudgetTypes || []
```

Comment:
```suggestion
    customer.planDetails?.availableReservedBudgetTypes || {}
```
`Object.values` takes in an object